pkg/login/social/social.go

```diff
@@ -54,9 +49,11 @@ type OAuthInfo struct {
 	TlsSkipVerify          bool
 }
 
-func (ss *SocialService) Init() error {
-	ss.oAuthProvider = make(map[string]*OAuthInfo)
-	ss.socialMap = make(map[string]SocialConnector)
+func ProvideService(cfg *setting.Cfg) *SocialService {
+	ss := SocialService{
+		oAuthProvider: make(map[string]*OAuthInfo),
+		socialMap:     make(map[string]SocialConnector),
```

Comment:
When running enterprise I panic on line 59, I think this should fix it.
```suggestion
		socialMap:     make(map[string]SocialConnector),
		Cfg: cfg,
```

Reply:
My latest commits should have resolved those. I just synced with latest main and resolved all conflicts, panics at startup and hopefully all tests are green again.

Reply:
FYI I think there were some misunderstandings between me and Arve earlier in regards to syncing with main and fixing all failing tests. Since a week back or so I told Arve that we'll handle all the syncs with main. This should not interfere with you fixing the Enterprise part given that your work is based on main branch compatible with this branch. Feel free to sync this with main whenever you want and/or ask @grafana/backend-platform for assistance.